x/epochs/types/epoch_info.go

```diff
@@ -21,9 +20,6 @@ func (ei *EpochInfo) EndEpoch() {
 
```

Comment:
Regarding
```
 // DEPRECATED: Identifier and Duration have been removed;
  reserved 1, 3;
  reserved "identifier", "duration";
```
@fedekunze, we still need `Duration` in `EpochInfo`. See one example above `ei.CurrentEpochStartTime = ei.CurrentEpochStartTime.Add(ei.Duration)`. I only deprecated `identifier`.